### DISCUSSION TITLE
TURN IN FILE RMD QUIZ  SEHINGGA TOMBOL MARK AS DONE JADI UNSUBMIT

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#531] [Q&A] [answered]
Saya telah menyelesaikan Quiz P4DS-PS dan File RMD nya saya turn in kan juga sehingga tombol Mark as Done nya berubah menjadi tombol Unsubmit, apakah ini tidak masalah ? (Karena belum klik tombol mark as Done)

<img width="885" alt="image" src="https://user-images.githubusercontent.com/123436963/216761527-a1f83144-9624-4276-9170-8710109a4304.png">

Terimakasih

**Comment by ramdhinafinita** [comment, marked as answer]
Dear Pak @Micoalgo 

Izin bant menjawab ya Pak, untuk kuis tersebut akan otomatis unsubmit atau ter `mark as done` jika bapak melampirkan file kedalam kuis tersebut. Jika tidak melampirkan, pilihan mark as done nya memang masih ada. Sehingga bapak tidak perlu khawatir karena kiri atas file lampiran bapak sudah `Turn in` yang artinya bapak sama dengan melakukan `Mark as Done`.

Semoga membantu

**Reply by Micoalgo** [reply]
Terimakasih Jawabannya

**Comment by Micoalgo** [comment]
Terimakasih jawabannya
